tests/release_scraper/test_release_scraper.py

```diff
@@ -7,127 +7,64 @@
 import os
 import urllib
 
+import pytest
+
 from mozdownload import ReleaseScraper
 from mozdownload.utils import urljoin
 
-import mozhttpd_base_test as mhttpd
-
-
-firefox_tests = [
-    # -p win32 -v 23.0.1
-    {'args': {'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.win32.exe',
-     'url': 'firefox/releases/23.0.1/win32/en-US/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win32 -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.win32.exe',
-     'url': 'firefox/releases/23.0.1/win32/en-US/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win64 -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'win64',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.win64.exe',
-     'url': 'firefox/releases/23.0.1/win64/en-US/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p linux -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'linux',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.linux.tar.bz2',
-     'url': 'firefox/releases/23.0.1/linux-i686/en-US/firefox-23.0.1.tar.bz2'},
-    # -a firefox -p linux64 -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'linux64',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.linux64.tar.bz2',
-     'url': 'firefox/releases/23.0.1/linux-x86_64/en-US/firefox-23.0.1.tar.bz2'},
-    # -a firefox -p mac -v 23.0.1
-    {'args': {'application': 'firefox',
-              'platform': 'mac',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.en-US.mac.dmg',
-     'url': 'firefox/releases/23.0.1/mac/en-US/Firefox 23.0.1.dmg'},
-    # -a firefox -p win32 -v 23.0.1 -l de
-    {'args': {'application': 'firefox',
-              'locale': 'de',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'filename': 'firefox-23.0.1.de.win32.exe',
-     'url': 'firefox/releases/23.0.1/win32/de/Firefox Setup 23.0.1.exe'},
-    # -a firefox -p win32 -v 23.0.1 --stub (old format)
```

Comment:
Similar to the other tests you worked on for the candidate scraper, please keep the comments about the old and new stub installer format.